airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/typing_deduping/SnowflakeDestinationHandler.java

```diff
@@ -361,4 +362,9 @@ private String toJdbcTypeName(final AirbyteProtocolType airbyteProtocolType) {
     };
   }
 
+  @Override
+  public void createNamespaces(@NotNull Set<String> schemas) {
+
```

Comment:
nit: is it safe to `throw new UnsupportedOperationException("not yet implemented")`? (just to be explicit that this _should_ be implemented, we just haven't gotten around to it yet

Reply:
yeah was planning to add that. just to be sure it doesn't get called erroneously.